src/fragments/lib/ssr/js/getting-started.mdx

```diff
@@ -1,3 +1,9 @@
+<Callout>
+
+SSR functionality in Amplify was primarily built for NextJS support and we cannot guarantee support on other frameworks. Additionally, it [does not work in a middleware context](https://github.com/aws-amplify/amplify-js/issues/9145) as it would in getServerSideProps.
```

Comment:
```suggestion
SSR functionality in Amplify was primarily built for Next.js support and we cannot guarantee support on other frameworks. Additionally, it [does not work in a middleware context](https://github.com/aws-amplify/amplify-js/issues/9145) as it would in getServerSideProps.
```

Reply:
Fixed this on the new wording as well